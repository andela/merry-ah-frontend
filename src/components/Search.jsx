import React, { Component } from 'react';

class Search extends Component {
    render() {
        return (
          <form className="form-inline my-2 my-lg-0 col-lg-7">
            <input type="search" placeholder="Search" aria-label="Search for"/>
          </form>
        )
      }
}

export default Search;