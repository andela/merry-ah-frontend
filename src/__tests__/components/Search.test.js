import React from 'react';
import { shallow } from 'enzyme';
import Search from '../../components/Search';

let wrapper;
beforeEach(() => {
 wrapper = shallow(<Search />);
})
describe('Search Component', () => {
  it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
  })
})
