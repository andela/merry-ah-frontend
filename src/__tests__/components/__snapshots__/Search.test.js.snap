// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Component should match snapshot 1`] = `
<form
  className="form-inline my-2 my-lg-0 col-lg-7"
>
  <input
    aria-label="Search for"
    placeholder="Search"
    type="search"
  />
</form>
`;
